lib/kontena/machine/vagrant/common.rb

```diff
@@ -0,0 +1,37 @@
+require 'fileutils'
+require 'erb'
+require 'open3'
+require 'json'
+
+module Kontena
+  module Machine
+    module Vagrant
+      module Common
+
+        def erb(template, vars)
+          ERB.new(template).result(OpenStruct.new(vars).instance_eval { binding })
+        end
+
+        def run_command(cmd)
+          exit_code = 0
+          Open3.popen3(cmd) do |stdin, stdout, stderr, wait|
```

Comment:
The weird `stdin` pipe seems to also cause issues: https://kontenacommunity.slack.com/archives/C1U90NQN8/p1499711369328277

> Vagrant is attempting to interface with the UI in a way that requires
> a TTY. Most actions in Vagrant that require a TTY have configuration
> switches to disable this requirement. Please do that or run Vagrant
> with TTY.